spec/features/student_loans_claim_spec.rb

```diff
@@ -1,124 +1,130 @@
 require "rails_helper"
 
 RSpec.feature "Teacher Student Loan Repayments claims" do
-  scenario "Teacher claims back student loan repayments" do
-    visit new_claim_path
-    expect(page).to have_text(I18n.t("student_loans.questions.qts_award_year"))
+  [
+    "enabled",
+    "disabled",
```

Comment:
Can we just use `true` and `false` here instead and pass the value straight to the `js:` in the scenario? The temporary variable feels surplus to requirements.

Reply:
I use that for the naming of the tests. It might make sense to reverse it though, and use `true` and `false` and then title the test depending on the value. I'll do that